examples/workflows/loop_workflow.rb

```diff
@@ -1,7 +1,13 @@
 require 'activities/hello_world_activity'
 
 class LoopWorkflow < Temporal::Workflow
-  def execute(count)
+  def execute(count, assert_memo)
+    assert_memo.each do |memo_key, memo_value|
```

Comment:
Not sure how you feel about this approach. Essentially, I need the workflow to check its own memo value on each iteration continues to match what was passed to the first iteration.

Reply:
I ended up scrapping this, having it return count and memo, and instead created a dedicated integration test for continue-as-new behavior. This also uncovered that memo was not being automatically copied to the next iteration as well, which is the behavior in Java and Go SDKs.